dist/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <title>Weather App</title>
+</head>
+<body>
+  <div id="content">
+    <div class="container my-5">
+      <h1 class="text-center title">Weather App</h1>
+      <form action="" class="search-location">
+        <input type="text" name="city"  autocomplete="off" placeholder="Add City" class="form-control text-muted form-rounded p-4 shadow-sm"
+        >
+      </form>
+      <div class="card rounded my-3 shadow-lg d-none back-card">
+        <div class="card-top text-center">
+          <div class="city-name my-3">
+            <p>Lagos</p>
+            <span>...</span>
+          </div>
+          <img src="./Assets/night-sky.svg" alt="" class="card-img-top time">
+        </div>
+        <div class="card-body">
+          <div class="card-mid row">
+            <div class="col-8 text-center temp">
+              <button  type= "button" class="btn btn-secondary"><span class="celsius">30&deg;C</span>
+            <span class="fahr d-none">30&deg;F</span></button>
+            
+
+            </div>
+            <div class="col-4 condition-temp">
+              <p class="condition">Thunder Storm</p>
+              <p class="high">30&deg;C</p>
+              <p class="low">27&deg;C</p>
+            </div>
+          </div>
+          <div class="icon-container card shadow mx-auto"> 
+            <img class="temp-icon" src="./Assets/weather-few-clouds-night.svg" alt="">
+          </div>
+          <div class="card-bottom px-5 py-4 row">
+            <div class="col text-center">
+              <p>30&deg;C</p>
```

Comment:
### [Optional]
Kindly remove this hardcoded Celcius that might confuse users from the actual data.